dotnetv3/S3/AbortMPUExample/AbortMPUExample/AbortMPU.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace AbortMPUExample
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.S3.Transfer;
+
+    /// <summary>
+    /// This example shows how to use the Amazon Simple Storage Service
+    /// (Amazon S3) to stop a multi-part upload process using the Amazon S3
+    /// TransferUtility. The example was created using the AWS SDK for .NET
+    /// version 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class AbortMPU
+    {
+        public static async Task Main()
+        {
+            string bucketName = "doc-example-bucket";
+
+            // If the AWS Region defined for your default user is different
+            // from the Regsion where your Amazon S3 bucket is located,
+            // pass the Region name to the S3 client object's constructor.
+            // For example: RegionEndpoint.USWest2.
+            IAmazonS3 client = new AmazonS3Client();
+
+            await AbortMPUAsync(client, bucketName);
+        }
+
+        /// <summary>
+        /// Aborts the multi-part copy process.
+        /// </summary>
+        /// <param name="client">The initialized client object used to create
+        /// the TransferUtility object.</param>
+        /// <param name="bucketName">The name of the multi-part copy operation
```

Comment:
The parameter name and comment aren't compatible.

Reply:
Changed to:

The name of the S3 bucket where the multi-part copy operation is in progress.